cedar-policy-core/src/est/expr.rs

```diff
@@ -923,6 +933,16 @@ impl TryFrom<&Node<Option<cst::Add>>> for Expr {
     }
 }
 
+fn is_add_valid_id(add: &cst::Add) -> Option<ast::Id> {
+    let cst::Name { path, name } = is_add_name(add)?;
+    if path.is_empty() {
+        let mut errs = ParseErrors::new();
+        name.to_valid_ident(&mut errs)
```

Comment:
This discards any `errs` generated by `.to_valid_ident()`.  Is this OK?

Reply:
I think so. These errors should be captured by the invalid attribute errors later on.